src/pages/account/Rewards/Rewards.tsx

```diff
@@ -80,6 +86,14 @@ export const Rewards: React.FC<{ delegator: string }> = ({ delegator }) => {
       key: 'createdTimestamp',
       render: (createdTimestamp: Date) => <Typography>{dayjs(createdTimestamp).format('YYYY-MM-DD HH:mm')}</Typography>,
     },
+    // {
```

Comment:
Remove it if don't need any more, otherwise and a TODO: message

Reply:
Ok, remove for now.